test/puppetlabs/puppetdb/fixtures.clj

```diff
@@ -17,20 +17,20 @@
             [puppetlabs.puppetdb.scf.migrate :refer [migrate!]]
             [puppetlabs.puppetdb.middleware :as mid]))
 
-(def ^:dynamic *db* nil)
+(def ^:dynamic *db* nil) ;; FIXME: perhaps this should be *db-spec* (cf db.clj)?
```

Comment:
OK, the context was that I think the new API may behave differently in some cases when given a an actual DB connection as compared to a config (i.e. spec).  For example, if I recall correctly, if you give with-db-connection a db, it won't open a new connection, it'll just use the one you give it (and close it), whereas "with-connection" (the old one) would create a new connection.  I believe that's what made me wonder.